### DISCUSSION TITLE
Demo Video

### DISCUSSION THREAD

**Discussion by growupanand** [growupanand/ConvoForm#274] [Ideas]
Please share any ideas regarding the demo video.

**Comment by growupanand** [comment]
@debangi29, Since you are making a demo video, and currently there is no demo video on the landing page. Are you intereseted in making one for the landing page?

**Reply by debangi29** [reply]
Yeah , sure i'll really like to work on that

**Reply by growupanand** [reply]
Great, So lets explore the product, and if you have any questions ask me. Then we can start working on this together.

**Comment by debangi29** [comment]
https://github.com/growupanand/ConvoForm/assets/117537653/6abead9d-3e5d-4fb6-af58-ba1d155c0b4f

Would this type of demo video work?
If any changes, please let me know!

**Reply by growupanand** [reply]
Nice, I think we can improve here,

- We need to use voice in video so people can understand better, we can plan a transcript before recording the video
- We have to add some zoom effect wherever we want to highlight user action, Example - https://x.com/growupanand/status/1746152838537814171
- Before stating demo video of App, we can first show some kind of slide where we can write some pain points which convoform can help them.


One more idea I have is like, we can show compare video with google forms. On the left side we will create a google form and on the right side we will create a convoform. So we can show that how easy is to create a form in convoform.


What do you think about these above suggestions?

**Comment by debangi29** [comment]
Yes, you are right , i thought about the voice and Google form comparison 
But I am unaware to add the zooming effect..
Can u please share?

**Reply by growupanand** [reply]
So that recording you see in the comment was done by a Mac OS app (https://www.screen.studio but it is not free), which automatically did that. But there is a free chrome extension which allow you to do same manually using a keyboard shortcut.

Try with this free tool - [Screenity - Screen Recorder & Annotation Tool](https://screenity.io/en/)

**Comment by debangi29** [comment]
I tried using the extension, but for some reason, I am not able to do the desired work
![Screenshot 2024-06-09 105447](https://github.com/growupanand/ConvoForm/assets/117537653/16968b31-bee4-491e-b008-6b1cfe570e71)


https://github.com/growupanand/ConvoForm/assets/117537653/7c6df9db-195d-408e-818a-81995b4e841b

I have added audio to the video, please check if it is fine
We need to make a 2-3 page slide adding a comparison with google form before the video and I guess it would be fine..!

**Comment by growupanand** [comment]
Audio is fine, we need to make a good transcript. Also, can we remove the watermark "veed"?

**Reply by growupanand** [reply]
Don't know about the extension error, can you check again with some other browser. I have tried that with chrome and it was working fine. May be some kind of permission issue for the extension.
